addons/bus/static/src/services/bus_service.js

```diff
@@ -29,6 +30,7 @@ export const busService = {
         const worker = new workerClass('/bus/websocket_worker_bundle', {
             name: 'SharedWorker' in window ? 'odoo:websocket_shared_worker' : 'odoo:websocket_worker',
         });
+        const currentUID = (Array.isArray(session.user_id) ? session.user_id[0] : session.user_id);
```

Comment:
Why 2 formats?

Reply:
In the frontend session info, it's a number while in the backend an array containing only one number..